test/it/oauth_id_site_token_grant_it.js

```diff
@@ -0,0 +1,32 @@
+
+var common = require('../common');
+var helpers = require('./helpers');
+var assert = common.assert;
+
+var stormpath = require('../../');
+
+describe('OAuthIdSiteTokenGrantAuthenticator',function(){
```

Comment:
Since this is a new file, could you add space between arguments, after `function`, and before `{`? This applies to the whole file.